app/assets/stylesheets/application.scss

```diff
@@ -23,3 +23,6 @@
 @import "tags-autocomplete";
 @import "registration";
 @import "text-extraction";
+
+// SUL Styles from the component library
+@import url("https://sul-dlss.github.io/component-library/styles/sul.css");
```

Comment:
This is not a stable URL. So the approach taken by #4539 will cause less unexpected change.

Reply:
Sounds good, part of why I wanted this as DRAFT is that I wasn't sure about the component library approach, so it makes sense to wait on your PR and rebase from that approach. Thanks.